testdata/src/embedded-func-pointer/embedded-func-pointer.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import "time"
+
+func main() {
```

Comment:
Should there be any assertions in this test?

Reply:
It's not a typical test. It's analyzed and the comments are turned into assertions. It's a bit odd but it was existing setup. 

I think I noticed one of the other tests also doesn't produce the output it claims to. If I spend much time on this in the future maybe I'll rework it.